exporter/info.go

```diff
@@ -274,47 +278,111 @@ func (e *Exporter) handleMetricsServer(ch chan<- prometheus.Metric, fieldKey str
 	}
 }
 
-func parseMetricsCommandStats(fieldKey string, fieldValue string) (string, float64, float64, error) {
+func parseMetricsCommandStats(fieldKey string, fieldValue string) (cmd string, calls float64, rejectedCalls float64, failedCalls float64, usecTotal float64, extendedStats bool, errorOut error) {
 	/*
-		Format:
-		cmdstat_get:calls=21,usec=175,usec_per_call=8.33
-		cmdstat_set:calls=61,usec=3139,usec_per_call=51.46
-		cmdstat_setex:calls=75,usec=1260,usec_per_call=16.80
-		cmdstat_georadius_ro:calls=75,usec=1260,usec_per_call=16.80
+		There are 2 formats. (One before Redis 6.2 and one after it)
+		Format before v6.2:
+			cmdstat_get:calls=21,usec=175,usec_per_call=8.33
+			cmdstat_set:calls=61,usec=3139,usec_per_call=51.46
+			cmdstat_setex:calls=75,usec=1260,usec_per_call=16.80
+			cmdstat_georadius_ro:calls=75,usec=1260,usec_per_call=16.80
+		Format from v6.2 forward:
+			cmdstat_get:calls=21,usec=175,usec_per_call=8.33,rejected_calls=0,failed_calls=0
+			cmdstat_set:calls=61,usec=3139,usec_per_call=51.46,rejected_calls=0,failed_calls=0
+			cmdstat_setex:calls=75,usec=1260,usec_per_call=16.80,rejected_calls=0,failed_calls=0
+			cmdstat_georadius_ro:calls=75,usec=1260,usec_per_call=16.80,rejected_calls=0,failed_calls=0
 
 		broken up like this:
 			fieldKey  = cmdstat_get
 			fieldValue= calls=21,usec=175,usec_per_call=8.33
 	*/
 
 	const cmdPrefix = "cmdstat_"
+	extendedStats = false
 
 	if !strings.HasPrefix(fieldKey, cmdPrefix) {
-		return "", 0.0, 0.0, errors.New("Invalid fieldKey")
+		errorOut = errors.New("Invalid fieldKey")
+		return
 	}
-	cmd := strings.TrimPrefix(fieldKey, cmdPrefix)
+	cmd = strings.TrimPrefix(fieldKey, cmdPrefix)
 
 	splitValue := strings.Split(fieldValue, ",")
-	if len(splitValue) < 3 {
-		return "", 0.0, 0.0, errors.New("Invalid fieldValue")
+	splitLen := len(splitValue)
+	if splitLen < 3 {
+		errorOut = errors.New("Invalid fieldValue")
+		return
 	}
 
 	calls, err := extractVal(splitValue[0])
```

Comment:
Don;'t you need to change this to `calls, err =` (instead of `:=`) so you don't shadow the `calls` variable?